docs/work/coverage-notes/README.md

```diff
@@ -0,0 +1,25 @@
+ 
+ # Introduction
+ When done correctly, coverage notes allow you to take the time you need off without resulting in any headaches for you or the team. In your coverage notes you should provide 3 things: What you are currently working on, what the status of your work is, and any other important notes that might be helpful to someone picking up the task. In conjunction with coverage notes, you should also talk to you PM(s) about anything else you might have missed. Your coverage notes should then be sent to your PM or most senior Dev.
```

Comment:
@seancdavis What would be the correct terminology for the end of this line "most senior Dev"? Would it be "Dev Lead", "Dev Project Lead", "other developers on the project" or something else?

Also, would it be sent to the PM/developers or just the PM in the project Slack channel? That way it's visible to everyone and I think we have a channel for every project.

Reply:
My gut here is that we set the precedent to post in the #general channel in Slack. That way everyone has visibility and there's less ambiguity about who should receive notes. What do you think?

Reply:
@seancdavis I think the #general channel on Slack is a good idea. Making it available to everyone was my goal with project-specific channels but #general gets everyone who needs it.

@J-Byron I think we could do something like "Your coverage notes should then be posted in the #general channel on Slack."